### PR TITLE
Maintenance updates for platform_view_swift

### PR DESCRIPTION
**Pull request by RedBrogdon**
This app was so old that I recreated the project structure from scratch, then moved code back into the individual files.

### CODE REVIEW

File: platform_view_swift/ios/Runner/AppDelegate.swift [hunk 1]
```diff
@@ -8,36 +8,36 @@ import Flutter
...
-      }
-    });
-
-    return super.application(application, didFinishLaunchingWithOptions: launchOptions)
-  }
-
-  func didUpdateCounter(counter: Int) {
-    flutterResult?(counter)
-  }
+    var flutterResult: FlutterResult?
```
Comment (domesticmouse) [inline platform_view_swift/ios/Runner/AppDelegate.swift]: I’m curious as to the source of the formatting

File: platform_view_swift/README.md [hunk 2]
```diff
@@ -1,10 +1,33 @@
...
+If you have a general question about Flutter, the best places to go are:
+
+* [The FlutterDev Google Group](https://groups.google.com/forum/#!forum/flutter-dev)
+* [The Flutter Gitter channel](https://gitter.im/flutter/flutter)
+* [StackOverflow](https://stackoverflow.com/questions/tagged/flutter)
+
+If you run into an issue with the sample itself, please file an issue
+in the [main Flutter repo](https://github.com/flutter/flutter/issues).
 
-pushd ios/ ; pod install ; popd
```
Comment (domesticmouse) [inline platform_view_swift/README.md]: Is this cocoa pods install no longer required?

File: platform_view_swift/ios/Runner/AppDelegate.swift (hunk 1 above)
Comment (RedBrogdon) [reply platform_view_swift/ios/Runner/AppDelegate.swift]: This is a community contribution from way back. I think the dev in question used soft tabs with stops at 2.

File: platform_view_swift/README.md (hunk 2 above)
Comment (RedBrogdon) [reply platform_view_swift/README.md]: No. There was a previous dependency on the Material components for iOS, which I pulled for simplicity's sake.

**Pull request merged**